db/schema.rb

```diff
@@ -154,6 +156,8 @@
     t.datetime "cgdub_dependencies_updated_at"
     t.text "gb_message"
     t.string "website"
+    t.string "git_access_key"
+    t.string "classroom_name"
```

Comment:
nit: would it be clearer to name this "git_classroom_name"?

Reply:
Good suggestion, fixed!